whelk-core/src/main/groovy/whelk/search/ESQuery.groovy

```diff
@@ -30,6 +30,13 @@ class ESQuery {
     private static final String OR_PREFIX = 'or-'
     private static final String EXISTS_PREFIX = 'exists-'
 
+    private static final Map recordsOverCacheRecordsBoost = [
+            'constant_score': [
+                    'filter': [ 'term': [ (JsonLd.RECORD_KEY + '.' + JsonLd.TYPE_KEY) : JsonLd.BASE_RECORD_TYPE ]],
+                    'boost': 1000.0
```

Comment:
I'm assuming this doesn't affect scoring in other ways (e.g. reaching a maximum making other scores moot)? (Otherwise we could perhaps "punish" use of `CacheRecord`?)

Reply:
It is not possible to specify negative boost values.